aptos_sdk/transactions.py

```diff
@@ -77,6 +77,16 @@ def prehash(self) -> bytes:
         hasher.update(b"APTOS::RawTransactionWithData")
         return hasher.digest()
 
+    @staticmethod
+    def deserialize(deserializer: Deserializer) -> RawTransactionWithData:
+        enum_type = deserializer.u8()
+        if enum_type == 0:
+            return MultiAgentRawTransaction.deserialize_inner(deserializer)
+        elif enum_type == 1:
+            return FeePayerRawTransaction.deserialize_inner(deserializer)
+        else:
+            raise Exception("Unhandled RawTransaction enum type")
```

Comment:
Slightly spooky, vs making use of bindings or whatever, but I guess the whole Python SDK is like this lol so it's alright. Since these magic numbers appear multiple times some constants could be good at least.